site/src/components/pagination.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Pagination
+description: Pagination related components in the Atlas Design System
+template: standard
+---
+
+# Pagination
+
+The pagination component consists of several elements:
+
+- `pagination-previous` and `pagination-next` for incremental navigation
+- `pagination-list` which displays page items for navigation to individual pages
+- `pagination-link` for page numbers and `pagination-ellipsis` for range numbers
+
+All elements are optional so you can compose your pagination as you wish.
+
+```html
+<nav class="pagination" role="navigation" aria-label="pagination">
+	<a class="pagination-previous" aria-label="previous">
+		<span class="icon" aria-hidden="true">
+			<span class="docon docon-arrow-left"></span>
+		</span>
+	</a>
+	<a class="pagination-next" aria-label="next">
+		<span class="icon" aria-hidden="true">
+			<span class="docon docon-arrow-right"> </span>
+		</span>
+	</a>
+	<ul class="pagination-list">
+		<li>
+			<a class="pagination-link" aria-label="Goto page 1">1</a>
+		</li>
+		<li>
+			<span class="pagination-ellipsis">&hellip;</span>
+		</li>
+		<li>
+			<a class="pagination-link" aria-label="Goto page 45">45</a>
+		</li>
+		<li>
+			<a class="pagination-link is-current" aria-label="Page 46" aria-current="page">46</a>
+		</li>
+		<li>
+			<a class="pagination-link" aria-label="Goto page 47">47</a>
+		</li>
+		<li>
+			<span class="pagination-ellipsis">&hellip;</span>
+		</li>
+		<li>
+			<a class="pagination-link" aria-label="Goto page 86">86</a>
+		</li>
+	</ul>
+</nav>
+```
+
+### Disabling links and specifying page numbers available
+
+You can disable some links if they are not active, or change the amount of page numbers available.
+
+```html
+<nav class="pagination" role="navigation" aria-label="pagination">
+	<a href class="pagination-previous" aria-label="previous" title="This is the first page" disabled>
+		<span class="icon" aria-hidden="true">
+			<span class="docon docon-arrow-left"></span>
+		</span>
+	</a>
+	<a href class="pagination-next" aria-label="next">
+		<span class="icon" aria-hidden="true">
+			<span class="docon docon-arrow-right"></span>
+		</span>
+	</a>
+	<ul class="pagination-list">
+		<li>
+			<a href class="pagination-link is-current" aria-label="Page 1" aria-current="page">1</a>
+		</li>
+		<li>
+			<a href class="pagination-link" aria-label="Goto page 2">2</a>
+		</li>
+		<li>
+			<a href class="pagination-link" aria-label="Goto page 3">3</a>
+		</li>
+	</ul>
+</nav>
+```
+
+## Modifiers
+
+### Current page
+
+The current page can be specified using the `is-current` modifier.
+
+### Positioning and ordering
+
+By default on tablet, the list is located on the left, and the previous/next buttons on the right.
+But you can change the positioning and order of these elements by using the `pagination-center` and `pagination-right` modifiers.
+
+#### Centered
+
+```html
+<nav class="pagination pagination-center" role="navigation" aria-label="pagination">
```

Comment:
Should the pagination next and previous buttons be close to the list? I wonder if we need to add a width atomic class for these examples. It looks quite odd when the next and previous buttons are nowhere close to the pagination list itself.
![image](https://user-images.githubusercontent.com/94572161/183221009-9c5e96f6-87a4-40ca-9662-0feac71cdfaa.png)
![image](https://user-images.githubusercontent.com/94572161/183221316-50574c42-2dd8-4691-bc39-4fc22d27d646.png)

Reply:
these were pulled from docs-ui, and didnt want to change it much if we do prob should make sure its ok first

Reply:
updated